### PR TITLE
change rails version to 5.0.1 exactly

### PR DESCRIPTION
**Pull request by Casey1449**
This bypasses a rails issue wherein our secret key must be 32 bytes. If you're just getting started on this repo, make sure to `bundle update` on your next pull to get on 5.0.1.

### CODE REVIEW

**Comment by Casey1449** [comment]
Closes #80, #81, #82

Everything appears to work smashingly except that the port proxying remains to be tested. 
(To test: Once we API-ify something the rails side, we'll make a call on the React side to make sure they communicate).

File: lib/tasks/start.rake [hunk 1]
```diff
@@ -0,0 +1,3 @@
+task :start do
+	exec 'foreman start -p 3000'
+end
```
Comment (chompasina) [inline lib/tasks/start.rake]: let's default to 2 spaces. :)

Comment (chompasina) [inline lib/tasks/start.rake]: okay you just showed me that it's actually 2, not 4. :) silly github.

**Comment by chompasina** [comment]
Looks like a good first step!

**Comment by Casey1449** [comment]
Also, now: Closes #79

**Comment by chompasina** [comment]
A note regarding `bundle update`: please don't ever run bundle update on a repo shared by multiple people. You'll want to check in with everyone first about updating because it can mess up dependencies, etc.

**Pull request merged**